libs/ui/src/styles.css

```diff
@@ -1,12 +1,26 @@
 @import './tailwind-generated.css';
 
+:root {
+  --gn-font-size-md: 1rem;
+  --gn-font-size-sm: 0.875rem;
+  --gn-title-color: #3B82F6;
+  --gn-link-color: #1D4ED8;
+}
+
 body {
   background-color: var(--color-background);
   color: var(--color-main);
 }
 a {
-  color: var(--color-primary);
+  color: var(--gn-link-color);
+}
+h1 {
+  color: var(--gn-title-color);
```

Comment:
I don't think it's the way to go.
You should, in the h1 of the CARD, use class `color-primary`, this way people will be able to change the color theme just by changing the --color-primary css variable.
We should have elements in the ui with primary and secondray colors (like h1, button, link etcc..) so the themes is customizable

Reply:
Maybe check https://tailwindcss.com/docs/customizing-colors#naming-your-colors and https://levelup.gitconnected.com/tailwindcss-with-css-variables-513abe2e9a5

Reply:
Ok, going to change it